firebase-firestore/src/test/java/com/google/firebase/firestore/local/SQLiteLocalStoreTest.java

```diff
@@ -460,18 +492,22 @@ public void testDoesNotAutoCreateIndexesWhenIndexLookUpIsExpensive() {
 
   @Test
   public void testIndexAutoCreationWorksWhenBackfillerRunsHalfway() {
-    Query query = query("coll").filter(filter("matches", "==", "foo"));
+    Query query =
```

Comment:
Can you explain the changes to `testIndexAutoCreationWorksWhenBackfillerRunsHalfway` also? Again, it appears that the code before and after _both_ test the scenario explained by the test method's name, but the "before" code is simpler.